Adding IBM Cloud installation docs (#5964)
kyma-project/kyma: docs/kyma/04-04-cluster-installation.md

>**CAUTION:** If you define your own Kubernetes jobs on the AKS cluster, follow [this](/components/service-mesh/#troubleshooting-kubernetes-jobs-fail-on-aks) troubleshooting guide to avoid jobs running endlessly on AKS deployments of Kyma.

  </details>
  <details>
  <summary label="IBM Cloud">
  IBM Cloud
  </summary>
  

> 1. Select a name for your cluster. Export the cluster name and the zone of type ```dc``` you want to deploy to as environment variables. Run:

Reviewer: ```suggestion
1. Select a name for your cluster. Export the cluster name, the zone of type `dc` you want to deploy to, and your IBM Cloud account email address as environment variables. Run:
```

    ```bash
    export CLUSTER_NAME={CLUSTER_NAME_YOU_WANT}

>     export CLUSTER_ZONE={ZONE_TO_DEPLOY_TO}

Reviewer: ```suggestion
    export CLUSTER_ZONE={ZONE_TO_DEPLOY_TO}
    export IBM_EMAIL={YOUR_IBM_CLOUD_ACCOUNT_EMAIL}
```

    ```

    >**TIP:** Run: ```ibmcloud ks supported-locations``` to list available zones. 

2. Create a cluster in the defined zone. Run:

    ```bash

>     ibmcloud ks cluster create classic --zone ${CLUSTER_ZONE} --machine-type b3c.4x16 --workers 3 --name ${CLUSTER_NAME} --public-service-endpoint

Reviewer: ```suggestion
    ibmcloud ks cluster create classic --zone $CLUSTER_ZONE --machine-type b3c.4x16 --workers 3 --name $CLUSTER_NAME --public-service-endpoint
```

    ```

3. Configure kubectl to use your new cluster. Run:

    ```bash

>     ibmcloud ks cluster config --cluster ${CLUSTERN_NAME}

Reviewer: ```suggestion
    ibmcloud ks cluster config --cluster $CLUSTER_NAME
```

    ```

>     >**TIP:** Copy and paste the export command that is displayed in your terminal to set the KUBECONFIG environment variable.

Reviewer: ```suggestion
4. Copy, paste, and run the export command that is displayed in your terminal to set the `KUBECONFIG` environment variable.
```

> 4. Add your account as the cluster administrator:

Reviewer: ```suggestion
5. Add the cluster administrator role to your user account:
```
Is it really an account you're adding or a role? What do you think about the suggested version?

    ```bash

>     kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=$[Your IBM Cloud account email]

Reviewer: ```suggestion
    kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=$IBM_EMAIL
```
